run/events/Startup.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+using System;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace HelloEvents
+{
+    public class Startup
+    {
+        public void ConfigureServices(IServiceCollection services)
+        {
+        }
+
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+        {
+            if (env.IsDevelopment())
+            {
+                app.UseDeveloperExceptionPage();
+            }
+
+            app.UseRouting();
+
+            app.UseEndpoints(endpoints =>

```

Comment:
I'd probably add a comment here for folks who've used ASP.NET Core in an MVC or WebAPI setting before, but haven't seen this way of responding to requests.

Reply:
What sort of comment you had in mind? In ASP.NET Core 3.1, this is the default template you get when you run `dotnet new web`

Reply:
Just something around "Sample responses to GET and POST requests" or the like.
I suspect most ASP.NET Core developers stick to MVC, Razor pages or WebAPI most of the time - this is probably lower level than they're used to.

Reply:
Ok, I added a couple minor comments